lib/vector-buffers/src/variants/disk_v2/tests/known_errors.rs

```diff
@@ -686,20 +700,6 @@ async fn reader_throws_error_when_record_is_undecodable_via_metadata() {
     static GET_METADATA_VALUE: AtomicU32 = AtomicU32::new(0);
     static CAN_DECODE_VALUE: AtomicU32 = AtomicU32::new(0);
 
-    impl AsMetadata for u32 {
```

Comment:
Clippy complained that the impl would be pulled up so better to define it outside.